### PR TITLE
Update to gradle 7

### PR DESCRIPTION
**Pull request by Yanpas**
Current script is incompatible with latest gradle.

Didn't test publising, but build works

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -114,51 +114,56 @@ artifacts {
...
+                releasesOnly()
+            }
+        }
+        maven {
+            url "https://oss.sonatype.org/content/repositories/snapshots/"
+            mavenContent {
+                snapshotsOnly()
+            }
+        }
+    }
```
Comment (dzmipt) [inline build.gradle:164]: you loose authentication details... Check in the old version there was
`authentication(userName: ossrhUsername, password: ossrhPassword)`
I think they should be somewhere here

**Review by dzmipt — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 1 above)
Comment (Yanpas) [reply build.gradle:164]: Will look at it. Are these properties required to be set (even if you are not going to publish anything) in $HOME/.gradle/gradle.properties?

**Comment by dzmipt** [comment]
Yes, the properties should be defined even for build or any other task.
People who are not publishing define any dummy values.

чт, 16 сент. 2021 г., 02:17 Yan Pashkovsky ***@***.***>:

**Comment by Yanpas** [comment]
Added gradle wrapper to VCS as it's common practise. This way we fix gradle version that everybody will use.

**Comment by dzmipt** [comment]
Thanks @Yanpas !

**Pull request merged**